maude.sublime-syntax

```diff
@@ -0,0 +1,107 @@
+%YAML 1.2
+---
+# See http://www.sublimetext.com/docs/syntax.html
+name: Maude
+file_extensions: [maude, ]
```

Comment:
Is there a reason for the empty element in this list?

Reply:
I think this was because of a habitual syntax style I use for lists in python.
Checking the [Sublime docs](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html) one exampleslook like this:
`file_extensions: [c, h]` 
so if you prefer changing the line to 
`file_extensions: [maude]`
that should be fine. 

Per [YAML  1.2.2 spec](https://yaml.org/spec/1.2.2/#flow-collection-styles):
> Flow collection entries are terminated by the “,” indicator. The final “,” may be omitted.

So I think 
`file_extensions: [maude]`
and 
`file_extensions: [maude, ]`
are equivalent.